Source/WebKit/WebProcess/WebCoreSupport/WebColorChooser.h

```diff
@@ -50,11 +40,19 @@ namespace WebKit {
 
 class WebPage;
 
-class WebColorChooser : public WebCore::ColorChooser, public CanMakeWeakPtr<WebColorChooser> {
+class WebColorChooser : public WebCore::ColorChooser, public RefCountedAndCanMakeWeakPtr<WebColorChooser> {
+    WTF_MAKE_TZONE_ALLOCATED_INLINE(WebColorChooser);
```

Comment:
I don't think this needs to be INLINE. We can probably just use `WTF_MAKE_TZONE_ALLOCATED` her and `WTF_MAKE_TZONE_ALLOCATED_IMPL` in the cpp like we usually do.

Reply:
I made it INLINE because the abstract base class `ColorChooser` is INLINE. Should I change the base class to `WTF_MAKE_TZONE_ALLOCATED` as well?

Reply:
The base class is probably inline because there is no CPP. As far as I know nothing prevents the base class from being inline and the subclass from being out of line. Therefore, I don't see why you'd need to modify the base class.

Reply:
Ok that makes sense. Fixed!